src/main/webapp/i18n/de/instructorDashboard.json

```diff
@@ -0,0 +1,11 @@
+{
+    "artemisApp": {
+        "instructorDashboard": {
+            "buildPlans": {
+                "delete": {
+                    "question": "Sollen build plans für Exercise {{ id }} wirklich dauerhaft gelöscht werden?"
```

Comment:
Translate "build plans" to "alle Studierenden Build Pläne" and "Exercise" to "Programmieraufgabe"

Reply:
Done.